src/slack/slack.js

```diff
@@ -621,10 +621,11 @@ export class Slack {
         const orderAttachment = this.userOrderAttachment(order, question)
         orderAttachment.actions = [additionalButton, CANCEL_ORDER_ACTION].filter(Boolean)
 
-        order.orderConfirmation = await this.boltApp.client.chat.postMessage({
+        const response = await this.boltApp.client.chat.postMessage({
           channel: userId,
           attachments: [orderAttachment],
         })
+        order.originalMessageInfo = (({channel, ts}) => ({channel, ts}))(response)
```

Comment:
oof, what a code 😄 it's definitely inovative, but I'd go with simpler:
```js
order.originalMessageInfo = {channel: response.channel, ts: response.ts}
```
or
```js
const {channel, ts} = response
order.originalMessageInfo = {channel, ts}
```
or when I'm thinking about it, you don't need `reponse` object:
```js
const {channel, ts} = await this.boltApp.client.chat.postMessage(...)
order.originalMessageInfo = {channel, ts}
```

Reply:
I found it in [StackOverflow](https://stackoverflow.com/questions/17781472/how-to-get-a-subset-of-a-javascript-objects-properties) 😄 

Thanks for the simple solution  👍